frameworks/mljarsupervised/setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+HERE=$(dirname "$0")
+#AMLB_DIR="$1"
+# by passing the module directory to `setup.sh`, it tells it to automatically create a virtual env under the current module.
+# this virtual env is then used to run the exec.py only, and can be configured here using `PIP` and `PY` commands.
+. $HERE/../shared/setup.sh $HERE
+#. $AMLB_DIR/frameworks/shared/setup.sh $HERE
+PIP install -r $HERE/requirements.txt
```

Comment:
I just pushed changes for `setup` recently to simplify version upgrade and to make it easier to benchmark multiple versions of the same framework.
Can you please change your `setup.sh` following the `autosklearn` example? https://github.com/openml/automlbenchmark/blob/master/frameworks/autosklearn/setup.sh

You should also be able to remove the `requirements.txt` then.

Reply:
Sure! `setup.sh` is changed